transport/http/inbound.go

```diff
@@ -124,6 +129,14 @@ func InboundTLSMode(mode yarpctls.Mode) InboundOption {
 	}
 }
 
+// InboundUseHTTP2 returns an InboundOption that sets whether the inbound should
+// accept HTTP/2 requests. If set to false, the inbound will only accept HTTP/1
+func InboundUseHTTP2(useHTTP2 bool) InboundOption {
```

Comment:
Maybe it's personal, but for me it sounds like "http2 is disabled by default". So if we want to enable it by default, it should be "DisableHTTP2", wdyt?

Reply:
yep, it make sense, but as we are introducing new support (http2) I thought options should be explicitly mentioning this usage. I can change this as I don't have strong opinion on this :)